### DISCUSSION TITLE
[FEATURE] 升级界面支持列出历史版本发行日志

### DISCUSSION THREAD

**Discussion by NHZEX** [1Panel-dev/1Panel#5137] [Ideas]
### 1Panel 版本

v1.10.7-lts

### 请描述您的需求或者改进建议

![image](https://github.com/1Panel-dev/1Panel/assets/14545600/9dc5512e-e741-484c-a33c-830ae0ce7524)

该升级界面支持查看当前运行版本相对最新版本的完整发行日志，方便快速审查。

### 请描述你建议的实现方案

1. 查看`1Panel`当前运行版本相对最新版本的完整发行日志。
2. 应用商店如果有条件也考虑支持，基于`github`这类开源托管有接口应该比较好获取.

**Comment by wanghe-fit2cloud** [comment]
1Panel 自身系统升级日志是保存在 `阿里 OSS` 中，不太容易列出相对完整的更新日志，因此该需求暂时只能到 `GitHub Release` 或者 `官方文档` 中去查看。

**Reply by wanghe-fit2cloud** [reply]
应用的升级记录也同理，可能部分应用可以通过接口获取，但是也不能保证完整性。
